### PR TITLE
Added ReactJS Tutorial under React Category

### PR DESCRIPTION
**Pull request by LaxmanJonchhen12**
Added ReactJS Tutorial for Beginners tutorials playlist for beginners by Codevolution

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -469,6 +469,8 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [React Patterns](https://reactpatterns.com/)
 - [Useful Packages you should know in React](https://medium.com/better-programming/14-useful-packages-every-react-developer-should-know-55b47a325d3)
 - [Learn React Test Driven Development](https://learntdd.in/react/)
+- [ReactJS Tutorial for Begiiners] (https://youtube.com/playlist?list=PLC3y8-rFHvwgg3vaYJgHGnModB54rxOk3)
```
Comment (shahednasser) [inline README.md:472]: Please fix the title as it has a typo. Also, remove the space between ] and ( as the link is now invalid.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi @LaxmanJonchhen12 thank you for your contribution. I have added a comment below.

File: README.md (hunk 1 above)
Comment (LaxmanJonchhen12) [reply README.md:472]: Sure will fix the error

**Comment by LaxmanJonchhen12** [comment]
@shahednasser  I have made the required changes.

**Comment by shahednasser** [comment]
Great thank you!

**Pull request merged**